### PR TITLE
Fix node.status.capacity.cpu conversion issue

### PR DESCRIPTION
**Pull request by GGGitBoy**
**Problem:**

Aliyun virtual kubelet node cannot be displayed in the Rancher UI because the node.status.capacity.cpu of the virtual kubelet is `1M`, which is not a number.

`1000m`, `1`, `100`, `1M` are all valid values for node.status.capacity.cpu field in Kubernetes.

However, in Rancher types, the filed is int, and we will not convert it to number if they are not int in types.

**Solution:**

So we need to handle other cases

**Related Issue:**

https://github.com/rancher/rancher/issues/26226

### CODE REVIEW

File: pkg/schemas/mapper/os_info.go [hunk 1]
```diff
@@ -15,16 +16,16 @@ func (o OSInfo) FromInternal(data map[string]interface{}) {
 	if data == nil {
 		return
 	}
-	cpuInfo := map[string]interface{}{
-		"count": values.GetValueN(data, "capacity", "cpu"),
+	cpuInfo := map[string]interface{}{}
+	cpuNum, err := resource.ParseQuantity(convert.ToString(values.GetValueN(data, "capacity", "cpu")))
+	if err == nil {
+		cpuInfo["count"] = cpuNum.Value()
 	}
```
Comment (gitlawr) [inline pkg/schemas/mapper/os_info.go:23]: Can we fall back on the original handling when the parse failed?

**Review by gitlawr — COMMENTED** [review]
Can you add some unit tests?

File: pkg/schemas/mapper/os_info.go (hunk 1 above)
Comment (GGGitBoy) [reply pkg/schemas/mapper/os_info.go:23]: Use `resource.ParseQuantity` can cover the original handling. 

before, we can handle values
memory: `Ki`
Cpu:  `“”`

Now we can handle all valid values in Kubernetes
memory: `Ki`, `Mi`, `Gi`, `Ti`, `Pi`, `Ei`
Cpu: `n`, `u`, `m`, `“”`, `k`, `M`, `G`, `T`, `P`, `E`

**Comment by GGGitBoy** [comment]
Update, add the unit tests

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by dramich — APPROVED** [review]

**Pull request merged**